source/fundamentals/builders.txt

```diff
@@ -7,6 +7,7 @@ Builders
 .. toctree::
 
    /fundamentals/builders/aggregates
+   /fundamentals/builders/aggregation-expression-operations
```

Comment:
Slating this for discussion with Dylan and Chris. Maybe we can potentially leave it here for now to get the page out, and then move it

Reply:
We can move out of builders, but I don't think it should be a sub-page of Aggregation. Perhaps, it can be its own item that comes sequentially after Aggregation.

Reply:
I think this looks ok, since it is somewhat in line with the CRUD Operations heading above...